packages/fakta-etablert-tilsyn/src/ui/components/beredskap/beredskapsperiode-vurderingsdetaljer/BeredskapsperiodeVurderingsdetaljer.tsx

```diff
@@ -1,9 +1,9 @@
 import { Box, Margin, DetailView, LabelledContent, LinkButton, AssessedBy } from '@navikt/ft-plattform-komponenter';
-import React, { useContext } from 'react';
+import React from 'react';
+import { useSaksbehandlerOppslag } from 'shared-components';
```

Comment:
Feil import her måske? Med mindre du har endret bort fra `@fpsak-frontend/shared-components` lenger nede i PR-en. Stay tuned for more!

Reply:
Ser ut til at du må oppdatere importen her, ja. Det gjelder flerer filer, så blir vel bare find og replace de stedene det gjelder

Reply:
hmm oi, brukte bare automatisk import i vscode